CHANGELOG.adoc

```diff
@@ -3,8 +3,53 @@
 
 = UNRELEASED
 
+== Dependencies
+
+=== Updated `+agent-rs+` to 0.20.0
+=== Updated `+candid+` to 0.7.15
+
 == DFX
 
+=== feat: support for configuring assets in assets canister
+- The `+.ic-assets.json+` file should be placed inside directory with assets, or it's subdirectories. Multiple config files can be used (nested in subdirectories). Example of `.ic-assets.json` file format:
+[source, json]
+----
+[
+    {
+        "match": ".*",
+        "cache": {
+            "max_age": 20
+        },
+        "headers": {
+            "X-Content-Type-Options": "nosniff"
+        },
+        "ignore": false
+    },
+    {
+        "match": "**/*",
+        "headers": null
+    },
+    {
+        "match": "file.json",
+        "ignore": true
+    }
+]
+----
+- Configuring assets works only during asset creation.
```

Comment:
Let's call out a a little more visibly that at present, changes to these fields don't take effect once the asset is created, and that we will follow up with improvements to handle updating these properties.